src/jit/codegenxarch.cpp

```diff
@@ -8027,6 +8059,19 @@ CodeGen::genPutStructArgStk(GenTreePtr treeNode, unsigned baseVarNum)
     assert(baseVarNum != BAD_VAR_NUM);
     
     var_types targetType = treeNode->TypeGet();
+
```

Comment:
I don't think that would be all that valuable, as it is a very small special case. Plus, I expect much of this code to be refactored in the near future in any case.